lib/active_model/serializer/adapter/json.rb

```diff
@@ -37,7 +37,7 @@ def serializable_hash(options = {})
             @result = @core.merge @hash
           end
 
-          { root => @result }
+          { (options[:root] || root) => @result }
```

Comment:
Thanks for this.

Would you mind including some more info about the bug you're fixing and adding a test?

Did you consider using `options.fetch(:root) { root }` or `options[:root] ||= root` or other ways of checking for a root option? (Curious what considerations led you to this approach)